NEWS.md

```diff
@@ -1,6 +1,6 @@
 ## v2.1.0 In progress
 ### Breaking changes
-* Description ([ISSUE_NUMBER](https://issues.folio.org/browse/ISSUE_NUMBER))
+* Delete PUT endpoint from authority-source-files api([MODELINKS-161](https://issues.folio.org/browse/MODELINKS-161))
```

Comment:
If it is breaking change we have to change version of the module to the next major one: v3.0.0